### PR TITLE
Support V4.0

### LINKED ISSUE #123: Support MongoDB v4
Just a feature request - thanks!

### PR DESCRIPTION
**Pull request by fllaca**
Continued the work of @ftx in #134 

Closes #123

### CODE REVIEW

File: tasks/install.debian.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 - name: Establish some role-related facts
   set_fact:
     mongodb_is_systemd: "{{ sbin_init.stat.islnk is defined and sbin_init.stat.islnk }}"
-    mongodb_major_version: "{{ mongodb_version[0:3] }}"
+    mongodb_major_version: "{{ mongodb_version[0:4] }}"
```
Comment (UnderGreen) [inline tasks/install.debian.yml]: Why did you change that slice here? So you changed, for example: `3.6.2` from `3.6` to `3.6.`

File: tasks/install.debian.yml [hunk 2]
```diff
@@ -19,21 +19,22 @@
 
 - name: Fail when used wrong mongodb_version variable with Debian Stretch
   fail:
-    msg: "mongodb_version variable should be '3.2' or '3.6' for Debian Stretch"
+    msg: "mongodb_version variable should be '3.2' or '3.6' or '4.0' for Debian Stretch"
```
Comment (UnderGreen) [inline tasks/install.debian.yml]: BTW 3.2 reached EOL last September. If you can get rid of it in this PR, it will be very appreciated.

File: tasks/install.debian.yml (hunk 1 above)
Comment (fllaca) [reply tasks/install.debian.yml]: This change was done by @ftx, but I think you are right. I'll change it back to `[0:3]`, unless @ftx gives a reason for this change.

File: tasks/install.debian.yml (hunk 2 above)
Comment (fllaca) [reply tasks/install.debian.yml]: Done!

File: tasks/install.debian.yml (hunk 1 above)
Comment (fllaca) [reply tasks/install.debian.yml]: Reverted this change.
